### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by ericgaspar**
- Upgrade to version 2.7.8
- install Redis-server package

### CODE REVIEW

**Comment by ericgaspar** [comment]
!testme

File: scripts/_common.sh [hunk 1]
```diff
@@ -4,7 +4,7 @@
 # COMMON VARIABLES
 #=================================================
 
-pkg_dependencies="g++ libjemalloc1|libjemalloc2 libjemalloc-dev zlib1g-dev libreadline-dev libpq-dev libssl-dev libyaml-dev libcurl4-dev libapr1-dev libxslt1-dev libxml2-dev vim imagemagick postgresql postgresql-server-dev-all postgresql-contrib optipng jhead jpegoptim gifsicle brotli"
+pkg_dependencies="g++ libjemalloc1|libjemalloc2 libjemalloc-dev zlib1g-dev libreadline-dev libpq-dev libssl-dev libyaml-dev libcurl4-dev libapr1-dev libxslt1-dev libxml2-dev vim imagemagick postgresql postgresql-server-dev-all postgresql-contrib optipng jhead jpegoptim gifsicle brotli redis-server"
```
Comment (JimboJoe) [inline scripts/_common.sh:7]: OK so now redis-server package is not installed by default in YunoHost, is it?

**Review by JimboJoe — APPROVED** [review]
LGTM 👍

File: scripts/_common.sh (hunk 1 above)
Comment (ericgaspar) [reply scripts/_common.sh:7]: holy cow! 🐮 you are right. Redis-server is preinstalled in YunoHost

**Pull request merged**